geospaas/vocabularies/tests.py

```diff
@@ -234,6 +445,8 @@ def setUp(self):
             for model in models]
 
     def test_command_update_vocabularies(self):
+        """Check that the command does not update the vocabularies if they are present"""
```

Comment:
@akorosov is this actually correct? The code is pretty convoluted.

Reply:
Yes, it is correct.